relay-replays/src/lib.rs

```diff
@@ -147,7 +147,7 @@ struct VersionedMeta {
 
 #[derive(Debug, Default, Deserialize, Serialize)]
 struct User {
```

Comment:
can we just use this struct here instead? https://github.com/getsentry/relay/blob/14d431fa029a25916220f3e193bcc3a6bebbc6e1/relay-general/src/protocol/user.rs#L42